src/main/java/gov/nih/nci/evs/api/service/AbstractStardogLoadServiceImpl.java

```diff
@@ -207,6 +192,13 @@ private void loadConceptsRealTime(
 
     if (CollectionUtils.isEmpty(allConcepts)) {
       logger.warn("Unable to load. No concepts found!");
+      Audit.addAudit(
+          operationsService,
+          "No concepts found",
+          "loadConceptsRealTime",
+          terminology.getTerminology(),
+          "No concepts found!",
+          "warning");
```

Comment:
you call this "log level" if we're using the value "ERROR" in other places, we should use the value "WARN" here instead of "warning"

Reply:
NOTE: there are other ones, I'm not going to mark them all.